src/main/java/org/openbase/bco/bcozy/util/Languages.java

```diff
@@ -0,0 +1,128 @@
+package org.openbase.bco.bcozy.util;
+
+import java.io.File;
+import java.net.URL;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * Class to hold available languages.
+ *
+ * @author vdasilva
+ */
+public class Languages {
+
+
+    /**
+     * Holder-Class for Threadsafe-Singleton.
+     */
+    private static final class InstanceHolder {
+        static final Languages INSTANCE = new Languages();
+    }
+
+    public static Languages getInstance() {
+        return InstanceHolder.INSTANCE;
+    }
+
+    private final Language english = fromLocale(Locale.US);
+
+    private final List<Language> languages;
+
+    /**
+     * Initializes Languages, either from available property-files or uses ["en_US","de_DE"].
+     */
+    private Languages() {
+        List<Locale> bundles = getLocalesFromBundles();
+        if (!bundles.isEmpty()) {
+            this.languages = bundles.stream().map(this::fromLocale).collect(Collectors.toList());
+        } else {
+            this.languages = Arrays.asList(english, fromLocale(Locale.GERMANY));
```

Comment:
Why is the fallback language forced to german?
Can this be replaced by `Locale.getDefault(`) ?